src/core/index.js

```diff
@@ -265,7 +265,7 @@ function extractClientBaseUrlOption(opts, domain) {
 
 export function extractTenantBaseUrlOption(opts, domain) {
   if (opts.configurationBaseUrl && typeof opts.configurationBaseUrl === "string") {
-    return opts.configurationBaseUrl;
+    return urljoin(opts.configurationBaseUrl, 'info-v1.js');
```

Comment:
now `configurationBaseUrl` is only the domain, it will append the blob endpoint.